tests/integration/test_backup_restore_on_cluster/test_cancel_backup.py

```diff
@@ -617,13 +617,13 @@ def test_error_leaves_no_trash():
 
 # A backup must be stopped if Zookeeper is disconnected longer than `failure_after_host_disconnected_for_seconds`.
 def test_long_disconnection_stops_backup():
+    create_and_fill_table(random_node(), num_parts=100)
+
     with NoTrashChecker() as no_trash_checker, ConfigManager() as config_manager:
         # Config "faster_zk_disconnect_detect.xml" is used in this test to decrease number of retries when reconnecting to ZooKeeper.
         # Without this config this test can take several minutes (instead of seconds) to run.
         config_manager.add_main_config(nodes, "configs/faster_zk_disconnect_detect.xml")
 
-        create_and_fill_table(random_node(), num_parts=100)
```

Comment:
This test uses settings from `configs/faster_zk_disconnect_detect.xml` to make ClickHouse stop retries faster after ZooKeeper disconnects. However such settings must be applied to `BACKUP`/`RESTORE` queries (which this tests checks), and not the `CREATE TABLE` queries which just create the initial tables.